src/app/resources/view-resources/resources-view.component.ts

```diff
@@ -182,4 +181,10 @@ export class ResourcesViewComponent implements OnInit, OnDestroy {
     this.fullView = this.fullView === 'on' ? 'off' : 'on';
   }
 
+  download(dataFile) {
+    const a = document.createElement('a');

```

Comment:
Same here.  You can simply create the link on the template to download the file.